packages/financial-templates-lib/src/price-feed/LPPriceFeed.ts

```diff
@@ -29,7 +70,7 @@ class LPPriceFeed extends PriceFeedInterface {
     blockFinder,
     minTimeBetweenUpdates = 60,
     priceFeedDecimals = 18,
-  }) {
+  }: Params) {
```

Comment:
again these arent auto assigned, though this is just a nit

Reply:
Same as above. I don't think it's possible (or at least I don't know the syntax) when the variables are passed as a single object param.